src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -222,16 +223,17 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
     val currentBranchName = intelliJ.project.getCurrentBranchName()
     Assert.assertEquals("develop", currentBranchName)
   }
-
-  private def saveThreadDumpToFile(): Unit = {
+*/
+  private def saveThreadDumpAndScreenshot(): Unit = {
     val pid = intelliJ.probe.pid()
+    intelliJ.probe.screenshot("exception")
     val threadStackTrace: String = Process("jstack " + pid) !!
-    val file: File = new File("build/thread-dump/thread_dump_" + pid + ".txt")
-    if (file.getParentFile.mkdirs()) {
-      val pw = new PrintWriter(file)
-      pw.write(threadStackTrace)
-      pw.close()
-    }
+    val artifactDirectory = "/tmp/ide-probe/screenshots/uiTest" + intelliJVersion + "/exception"
```

Comment:
Oops
![image](https://user-images.githubusercontent.com/3383210/173857188-31fb8f7b-f6af-41c5-b01a-53fdb58471b0.png)

Reply:
yes I change it now 😂